### ISSUE TITLE
model and weights where to find?

### ISSUE THREAD

**Issue by jinjunfei** [marinbenc/medical-polar-training#1]
After training the network, where are the results saved? What model to choose for testing? What are weights?

**Comment by marinbenc** [comment]
Hi, you can find the trained weights in the releases section here:

https://github.com/marinbenc/medical-polar-training/releases/tag/v1.0

While training, by default the checkpoints are saved inside `logs/` with a folder named after the current date, however you can set this with the `--logs` parameter.

You can choose a trained checkpoint and an appropriate model type for that checkpoint for testing.

I hope this helps. let me know if you have any other questions.

**Comment by jinjunfei** [comment]
Thank you very much for your reply, the previous question I have made clear. Another problem is that your trained weight model includes: "polyp_unet_polar_aug, polyp_unet_polar, polyp_unet_non_polar, polyp_stacked_hourglass_sigma_8_non_aug, polyp_stacked_hourglass_sigma_8." "I know from readme.md that train.py used to train Cartesian and pole networks, train_hourglass.py used to train center point prediction networks. I want to know how to set the parameters to get the above five different model files, and which of these five model files should be selected as the final model? We look forward to your reply, thank you very much!

**Comment by marinbenc** [comment]
From our experiments, the best models were when we combined a stacked hourglass model (with augmentation) with a polar U-Net with agumentation. In your case, that would mean using `polyp_stacked_hourglass_sigma_8` to find centerpoints, and then use those centerpoints to transform the input image to polar coordinates, and then give that image to `polyp_unet_polar_aug` to perform the segmentation.

You can see this approach in [test_centerpoint_model.py](https://github.com/marinbenc/medical-polar-training/blob/main/test_centerpoint_model.py).

I will direct you to [our paper](https://ieeexplore.ieee.org/document/9551998) for more details.

> For all networks, we use the Adam optimizer with a learning rate of 10e-3. A batch size of 8 was used for all networks except the center-point model, where a batch size of 6 was used for the lesion and liver datasets, and 8 for all remaining datasets. We trained all models up to a maximum of 200 epochs and used checkpoints after each epoch to store the model with the best validation loss.

**Comment by jinjunfei** [comment]
Thank you very much!
